documentation/2_Context_of_the_International_Data_Spaces/2_5_Meaningful_data.md

```diff
@@ -0,0 +1,15 @@
+
+The ability of IT systems to exchange data with unambiguous, shared meaning is called semantic interoperability. Semantic interoperability is an essential requirement for federated data networks such as IDS. It requires that data providers and data consumers in the network express their data offering or need using explicit reference to a common vocabulary. The commonality is important; if data is provided with references to a vocabulary that the receiving party is unfamiliar with, then the need to spend integration effort on their end remains. Vocabularies become common (shared) through the process of standardization.
+
+Many industries and other business ecosystems have turned to open standardization as a means to achieve semantic interoperability between its members. Open standardization means that members collaboratively maintain and develop semantic standards. It is a continuous balancing act between the need for strict uniformity to keep data consistent and easy to understand, and the need to accomodate for the fact that different organizations have different requirements for their data. 
+
+This means there is often a limit to the level of semantic interoperability that can be achieved. Every member in a business ecosystem operates with a different world view. These differences arise from operating in different jurisdictions, in different domains, carrying out different business processes, serving different marktes, offering different services, and so on. High variety between members in business ecosystems means that any semantic standard for that community will have to allow for flexibility, which means some integration effort will remain necessary. Low variety allow for stricter semantic standards that bring more uniformity, allowing for more efficient data sharing and automatisation.  
+
+In any case, governance has to be put in place to make sure that a semantic standard serves the needs of the community as best as possible and will remain doing so. How these governance processes can be organized is discussed in section 4.  
+
+From a technology perspective, achieving semantic interoperability requires the use semantic technologies to create Linked Data. Semantic technologies such as OWL, RDF and SKOS allow us to enrich data with meaning by creating links to other datasets and vocabularies, enabling automatic reasoning over data through rules. The role of vocabularies and other semantic technologies in IDS is discussed in section 3. 
```

Comment:
```suggestion
From a technology perspective, achieving semantic interoperability requires the use semantic technologies to create Linked Data. Semantic technologies such as RDF, SHACL, OWL and SKOS allow us to enrich data with meaning by creating links to other datasets and vocabularies, enabling automatic reasoning over data through rules. The role of vocabularies and other semantic technologies in IDS is discussed in section 3.
```
* added SHACL, which is also a relevant standard for semantic interoperability in the IDS.
* tried to order more logically, starting with RDF, which is the foundation of everything else.

Reply:
the use semantic technologies -> the use of semantic technologies